### PR TITLE
feat(icon): update "new", "active", "inactive", and "status" icons

### PR DESCRIPTION
**Pull request by jagarnica**
## Summary
- Updates the icons: new, active, inactive, status; to a new visual design. 

## Screenshots (if appropriate):
Screenshot 2023-09-01 at 10 59 38 AM
Screenshot 2023-09-01 at 10 59 43 AM
Screenshot 2023-09-01 at 10 59 48 AM
Screenshot 2023-09-01 at 10 59 53 AM

## Testing approaches

Locally running unit, component, and SSR tests.

### CODE REVIEW

**Review by vroske-ld — APPROVED** [review]

File: .changeset/ninety-ravens-argue.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@launchpad-ui/icons': minor
```
Comment (Niznikr) [inline .changeset/ninety-ravens-argue.md:2]:
```suggestion
'@launchpad-ui/icons': patch
```
Non-breaking `0.x` versioning changes

**Review by Niznikr — CHANGES_REQUESTED** [review]
The lockfile changes seem undesired. What version of pnpm are you using?

**Comment by jagarnica** [comment]
> The lockfile changes seem undesired. What version of pnpm are you using?

I was using an earlier version at first but I then upgraded to 8.7.1. I was on v7 but I had some issues with it

**Comment by Niznikr** [comment]
Could you revert the changes to the lockfile?

**Comment by jagarnica** [comment]
> Could you revert the changes to the lockfile?

Absolutely! I think I found how I caused that. I just reverted those changes now.

**Review by Niznikr — APPROVED** [review]

**Pull request merged**